actionview/lib/action_view/log_subscriber.rb

```diff
@@ -19,7 +19,16 @@ def render_template(event)
         message << " (#{event.duration.round(1)}ms)"
       end
     end
-    alias :render_partial :render_template
+
+    def render_partial(event)
+      info do
+        message = "  Rendered #{from_rails_root(event.payload[:identifier])}"
+        message << " within #{from_rails_root(event.payload[:layout])}" if event.payload[:layout]
+        message << " (#{event.duration.round(1)}ms)"
+        message << " #{cache_message(event.payload)}" if event.payload[:cache_partial]
+        message
```

Comment:
I don't see `log_payload` referenced here, so how does this tie in with the Action View accessor you've added?